translations/README.russian.md

```diff
@@ -0,0 +1,199 @@
+# Hasura GraphQL Engine
+
+[![Docs](https://img.shields.io/badge/docs-v1.0-brightgreen.svg?style=flat)](https://docs.hasura.io)
+[![CircleCI](https://circleci.com/gh/hasura/graphql-engine.svg?style=shield)](https://circleci.com/gh/hasura/graphql-engine)
+
+
+<a href="https://discord.gg/vBPpJkS"><img src="https://img.shields.io/badge/chat-discord-brightgreen.svg?logo=discord&style=flat"></a>
+<a href="https://twitter.com/intent/follow?screen_name=HasuraHQ"><img src="https://img.shields.io/badge/Follow-HasuraHQ-blue.svg?style=flat&logo=twitter"></a>
+<a href="https://eepurl.com/dBUfJ5"><img src="https://img.shields.io/badge/newsletter-subscribe-yellow.svg?style=flat"></a>
+
+Hasura GraphQL Engine - это сверхбыстрый графический сервер позволяющий создавать **мгновенно GraphQL API на основе Postgres**, с [**webhook triggers**](../event-triggers.md), и [**remote schemas**](../remote-schemas.md) для бизнес-логики.
+
+Hasura помогает стройть GraphQL приложения при поддержке Postgres или постепенно перейти на GraphQL для уже существующих приложений, использующих Postgres.
+
+Подробнее на [hasura.io](https://hasura.io) и [docs](https://docs.hasura.io).
+
+------------------
+
+![Hasura GraphQL Engine Demo](../assets/demo.gif)
+
+------------------
+
+![Hasura GraphQL Engine Realtime Demo](../assets/realtime.gif)
+
+-------------------
+
+## Характеристики
+
+* **Делать мощные запросы**: встроенная фильтрация, нумерация страниц, поиск шаблонов, объёмная вставка, обновление, удаление мутаций.
+* **Режим реального времени**: Конвертируйте любой запрос GraphQL в запрос в режиме реального времени, используя подписку.
+* **Слияние удаленных схем**: Доступ к пользовательским схемам GraphQL для бизнес-логики через единую конечную точку GraphQL Engine. [**Подробнее**](../remote-schemas.md).
+* **Запуск webhook-ов или serverless функций**: В ответ на события Postgres insert/update/delete.([ подробнее](../event-triggers.md))
+* **Работает с существующими базами данных**: Направьте его на существующую базу данных Postgres, чтобы мгновенно получить готовый к использованию GraphQL API.
+* **Подробный контроль доступа**: Динамический контроль доступа, интегрируемый с Auth системой (напр: auth0, firebase-auth).
+* **Высокая производительность и малое воздействие**: ~15MB изображение docker-a; ~50MB RAM @ 1000 req/s; многоядерный.
+* **Интерфейс администрирования и миграции**: Интерфейс администрирования и миграция схемы на Rails
+* **Postgres** ❤️: Поддержка Postgres типов (PostGIS/geo-location, и т.д.), преобразуется в *graphs*, запускает процедуры или функции, сохраненные посредством мутаций.
+
+Подробнее на [hasura.io](https://hasura.io) и [docs](https://docs.hasura.io).
+
+## Содержание
+<!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
+**Содержание**
+
+- [Быстрый запуск:](#быстрый_запуск)

```

Comment:
Some of the links in the table of contents are still broken.

Reply:
Done!

Reply:
The very first link in the table of contents is still not working. Everything else works 🙂

Reply:
Fixed the 1st link too. What silly mistakes I do today!:)